stone/backends/js_helpers.py

```diff
@@ -49,11 +49,11 @@ def fmt_obj(o):
         return json.dumps(o, indent=2)
 
 
-def fmt_error_type(data_type):
+def fmt_error_type(data_type, wrap_error_in=''):
     """
     Converts the error type into a JSDoc type.
     """
-    return 'Error.<%s>' % fmt_type(data_type)
+    return '%s.<%s>' % ((wrap_error_in if (wrap_error_in != '') else 'Error'), fmt_type(data_type))
```

Comment:
nit: And then update this to: `return '%s.<%s>' % (wrap_error_in, fmt_type(data_type))`